### DISCUSSION TITLE
4.6.19版本二维码登陆显示版本过低

### DISCUSSION THREAD

**Discussion by GalaxyeyesTchyonDragon** [ywmoyue/biliuwp-lite#383] [bug]
### Bug 描述

4.6.19二维码登陆显示版本过低

### 操作步骤

```plain text
二维码扫描无法登录
```

### 期望结果

希望回答解决

### 应用来源

Github/其他Github镜像站

### 设备类型

笔记本

### 设备架构

x64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
可以发一下日志吗？我这里没有复现出错误

**Reply by GD-Slime** [reply]
是会在哪里显示版本过低呢？电脑上还是手机上？
您的手机app版本是什么？
我这里实在复现不出来。
条件：Win11 x64 最新dev编译 手机bilibili客户端7.41.0

**Reply by GalaxyeyesTchyonDragon** [reply]
我这是Win10，手机版本是7.52.0

**Reply by GD-Slime** [reply]
我用手机7.52版本又试了一下，依旧不能复现错误。。
日志看的话也很奇怪，你的短信登录是否也不可用？

@ywmoyue 你那里可以尝试复现一下吗？

**Reply by ywmoyue** [reply]
刚刚那个日志含登录流程可能有敏感信息，我删楼层了

**Reply by ywmoyue** [reply]
我这边测了安卓概念版7.38.0和安卓7.17.0都没复现

**Reply by ywmoyue** [reply]
@GalaxyeyesTchyonDragon 你这边用短信验证码可以登录吗

**Reply by GalaxyeyesTchyonDragon** [reply]
![Screenshot_20231030_161716](https://github.com/ywmoyue/biliuwp-lite/assets/105159062/b8a9f615-9787-49ea-ac94-195136c814f4)
![Screenshot_20231030_161706](https://github.com/ywmoyue/biliuwp-lite/assets/105159062/34430801-9b68-4e0c-ab14-25406189639d)

**Comment by GalaxyeyesTchyonDragon** [comment]
短信验证码也不行，一直说我验证码错误

**Comment by GalaxyeyesTchyonDragon** [comment]
![屏幕截图 2023-10-30 161945](https://github.com/ywmoyue/biliuwp-lite/assets/105159062/a1977fd0-8d05-45e6-a53a-94e2099d26a9)

**Comment by GD-Slime** [comment]
@ywmoyue  还有一种可能性：
以前我记得有个api，在某个cdn节点上其风控就会更严格，其他cdn就没事。。 这次是否也是这样的原因？不然也想不出其他原因了。

**Comment by ywmoyue** [comment]
我刚新注册了个账号复现了
@GalaxyeyesTchyonDragon 你的b站账号是什么时候注册的，是否通过了入站考试

**Reply by GalaxyeyesTchyonDragon** [reply]
我的账号用了五年了，是六级

**Comment by GalaxyeyesTchyonDragon** [comment]
我退后版本也不行，实在没法子了

**Reply by GD-Slime** [reply]
#366 稍后可能会出cookie登录的后备选项，不行到时候再用cookie手动登录试一下吧

**Reply by GalaxyeyesTchyonDragon** [reply]
已经可以了，辛苦了

**Comment by ywmoyue** [comment]
@GalaxyeyesTchyonDragon   4.6.20已发布，麻烦试一下

**Reply by GalaxyeyesTchyonDragon** [reply]
已经可以了，辛苦了
